python/lsst/meas/astrom/denormalizedMatches.py

```diff
@@ -0,0 +1,58 @@
+import lsst.afw.table
+
+__all__ = ["matchesToCatalog"]
+
+def matchesToCatalog(matches, matchMeta=None):
+    """Write matches as a Catalog
```

Comment:
This could be more descriptive. It doesn't actually write.  How about "Join reference and source catalogs to associative* match list"

* your favorite of associative, cross-reference, junction.